### PR TITLE
Added Gradle Arch Linux installation and auto deployment using Gradle

### PR DESCRIPTION
**Pull request by jmromanos**

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -88,3 +96,104 @@ building the code with Travis if commits were added before setting up Travis:
...
+cargoStartLocal.dependsOn assemble
+cargoDeployRemote.dependsOn assemble
+
+cargo {
+  containerId='tomcat8x'
+  
+  remote {
+    hostname = 'valor.humiltat.es'
+    username = 'seny'
+    password = 'imparapla'
```
Comment (fjlopez) [inline description.md]: Risky! Passwords in clear in a public GitHub repository!!!

**Comment by fjlopez** [comment]
I think that you deserve the :gift: but you must fulfil one more task. Your solution is risky as `build.gradle` contains the password in clear. You should investigate how to use Travis support for [encrypted keys](https://docs.travis-ci.com/user/encryption-keys/).

**Comment by jmromanos** [comment]
Done @fjlopez

**Comment by fjlopez** [comment]
Explain what was added to the travis file automatically with the encryption command and done! You has gained a :gift:

**Comment by jmromanos** [comment]
Added :D @fjlopez

**Pull request merged**